src/ansible-doc.ts

```diff
@@ -0,0 +1,160 @@
+const allPluginsGenerator = (
+  suggestOptions?: Partial<Fig.Suggestion>
+): Fig.Generator => ({
+  script: "ansible-doc --list --json 2>/dev/null",
+  postProcess: function (output) {
+    const plugins = JSON.parse(output);
+    return Object.keys(plugins).map((key) => ({
+      name: key,
+      description: plugins[key],
+    }));
+  },
+});
```

Comment:
I mean, can we just refactor to this?
```suggestion
const allPluginsGenerator: Fig.Generator = {
  script: "ansible-doc --list --json 2>/dev/null",
  postProcess: function (output) {
    const plugins = JSON.parse(output);
    return Object.keys(plugins).map((key) => ({
      name: key,
      description: plugins[key],
    }));
  },
};
```

Reply:
I've committed this, but, it's failing `typecheck`. I don't know enough TypeScript to know what to do, can you help?